### PR TITLE
Add a new Chapter FAQ

### PR DESCRIPTION
**Pull request by fkaminski**
The FAQ's for starters are unsorted within one index page (for now). 
Please review the content carefully. Some of them are i.a. just copied form community entries ...

### CODE REVIEW

File: docs/manual/faq/_index.de.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Hier findest du eine Sammlung der häufigsten Fragen mit entsprechenden Lösungen. 
+
+Wenn du selbst einen Vorschlag für diesen Bereich hast, verwende den Link »Diese Seite bearbeiten« oben rechts. 
+Besitzt du ein GitHub-Konto und bist angemeldet erstellt GitHub automatisch einen Fork in dem du 
+deine Vorschläge hinzufügen kannst. Anschließend kannst du über GitHub einen Pull-Request erstellen.
+</br></br>
+
+{{% expand "Ich habe mein Administrator-Passwort vergessen." %}}
+Entferne hierzu in der Tabelle »tl_user« deiner Datenbank in der Spalte »admin« den Wert »1«. Anschließend kannst
```
Comment (xchs) [inline docs/manual/faq/_index.de.md]: BTW: Falls es in der Tabelle »tl_user« mehrere Datensätze gibt, bei denen das Admin-Flag gesetzt ist, müsste man den Wert zunächst bei allen zurücksetzen, um anschließend im Contao Installtool einen neuen Administrator anlegen zu können.

File: docs/manual/faq/_index.de.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+{{% expand "Wozu benötige ich den Contao Manager?" %}}
+Alle Informationen findest du unter [Über den Contao Manager](/de/installation/contao-manager/).
+{{% /expand %}}
+
+{{% expand "Wie kann man den Contao Manager aktualisieren?" %}}
+Der [Contao Manager](/de/installation/contao-manager/#haeufige-fragen-zum-contao-manager) führt beim Aufruf automatisch 
+im Hintergrund eine Prüfung durch. Sollte eine neue Version verfügbar sein, aktualisiert sich der Contao Manager selbst.
+{{% /expand %}}
+
+{{% expand "Was ist der Composer Resolver Cloud?" %}}
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
{{% expand "Was ist die Composer Resolver Cloud?" %}}
```

File: docs/manual/faq/_index.de.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+Der [Composer Resolver Cloud](https://composer-resolver.cloud/) erlaubt die Installation von Composer-Abhängigkeiten
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
Die [Composer Resolver Cloud](https://composer-resolver.cloud/) erlaubt die Installation von Composer-Abhängigkeiten 
```

File: docs/manual/faq/_index.de.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+du im Anschluss in der »Systemwartung« den Scriptcache leeren.
+{{% /expand %}}
+
+{{% expand "Kann ich den Contao Manager einer bestehenden Installation hinzufügen?" %}}
+Ja. Der [Contao Manager](/de/installation/contao-manager/#kann-der-contao-manager-zu-einer-bestehenden-installation-hinzugefuegt-werden) 
+erkennt bei der Installation deine bestehende Contao Installation.
+{{% /expand %}}
+
+{{% expand "Wozu benötige ich den Contao Manager?" %}}
+Alle Informationen findest du unter [Über den Contao Manager](/de/installation/contao-manager/).
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
Du benötigst den Contao Manager um Contao und Erweiterung zu installieren/aktualisieren/deinstallieren. Weiterführende Informationen findest du unter [Über den Contao Manager](/de/installation/contao-manager/).
```

File: docs/manual/faq/_index.de.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+gekennzeichnet ist. 
+{{% /expand %}}
+
+{{% expand "Kann man die URL Suffix ».html« entfernen?" %}}
+Du kannst in der [config.yml](/de/system/einstellungen/#config-yml) den Eintrag `url_suffix: ''` hinzufügen. 
+Anschließend musst du über den Contao-Manager (»Systemwartung« > »Prod.-Cache erneuern«) oder über die Konsole 
+einmalig den Anwendungs-Cache leeren.
+{{% /expand %}}
+
+{{% expand "Änderungen an meinen SCSS-Dateien werden nicht übernommen." %}}
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
{{% expand "Warum werden Änderungen an meinen SCSS-Dateien nicht übernommen?" %}}
```

File: docs/manual/faq/_index.de.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
+---
+title: "Fragen und Antworten"
+description: "Antworten zu den häufigsten Fragen."
+url: "faq"
+aliases:
+    - /de/faq/
+weight: 110
+---
+
+Hier findest du eine Sammlung der häufigsten Fragen mit entsprechenden Lösungen.
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
Hier findest du eine Sammlung der häufigsten Fragen mit den passenden Antworten. 
```

File: docs/manual/faq/_index.de.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+---
+
+Hier findest du eine Sammlung der häufigsten Fragen mit entsprechenden Lösungen. 
+
+Wenn du selbst einen Vorschlag für diesen Bereich hast, verwende den Link »Diese Seite bearbeiten« oben rechts. 
+Besitzt du ein GitHub-Konto und bist angemeldet erstellt GitHub automatisch einen Fork in dem du 
+deine Vorschläge hinzufügen kannst. Anschließend kannst du über GitHub einen Pull-Request erstellen.
+</br></br>
+
+{{% expand "Ich habe mein Administrator-Passwort vergessen." %}}
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
{{% expand "Ich habe mein Administrator-Passwort vergessen, wie kann ich es zurücksetzen?" %}}
```

File: docs/manual/faq/_index.de.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+Falls es in der Tabelle »tl_user« mehrere Datensätze gibt, bei denen das Admin-Flag gesetzt ist, kannst du den Wert
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
Falls es in der Datenbank-Tabelle »tl_user« mehrere Datensätze gibt, bei denen das Admin-Flag gesetzt ist, kannst du den Wert 
```

File: docs/manual/faq/_index.de.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+deine Vorschläge hinzufügen kannst. Anschließend kannst du über GitHub einen Pull-Request erstellen.
+</br></br>
+
+{{% expand "Ich habe mein Administrator-Passwort vergessen." %}}
+Falls es in der Tabelle »tl_user« mehrere Datensätze gibt, bei denen das Admin-Flag gesetzt ist, kannst du den Wert 
+zunächst bei allen zurücksetzen, um anschließend im Contao [Install-Tool](/de/installation/contao-installtool/) 
+einen neuen Administrator anzulegen.
+{{% /expand %}}
+
+{{% expand "Ich habe das Install-Tool Passwort vergessen." %}}
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
{{% expand "Ich habe das Install-Tool Passwort vergessen, wie kann ich es zurücksetzen?" %}}
```

File: docs/manual/faq/_index.de.md [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+{{% expand "Wie kann ich alle Variablen meines Templates anzeigen?" %}}
+Die Information hierzu findest du unter [Template-Daten anzeigen](/de/templates/data/).
+{{% /expand %}}
+
+{{% expand "Wie kann ich ein Insert-Tag in meinem Template verwenden?" %}}
+Zur Nutzung eines [Insert-Tag](/de/artikelverwaltung/insert-tags/) `{{date}}` in deinem Template muss du dieses 
+über `$this->replaceInsertTags('{{date}}')` einsetzen.
+{{% /expand %}}
+
+{{% expand "Es wird keine E-Mail über mein Formular versendet." %}}
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
{{% expand "Es wird keine E-Mail über mein Formular versendet, was muss ich machen?" %}}
```

File: docs/manual/faq/_index.de.md [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+Überprpüfe in der `parameters.yml` die [SMTP-Angaben](/de/system/einstellungen/#smtp-versand) deines Hosters oder 
+füge diese hinzu. Anschließend musst du über den Contao-Manager (»Systemwartung« > »Prod.-Cache erneuern«) oder über die 
+Konsole einmalig den Anwendungs-Cache leeren.
+{{% /expand %}}
+
+{{% expand "Wie kann ich Sprachenkürzel der URL hinzufügen" %}}
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
{{% expand "Wie kann ich das Sprachenkürzel der URL hinzufügen?" %}}
```

File: docs/manual/faq/_index.de.md [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+Du kannst in der [config.yml](/de/system/einstellungen/#config-yml) den Eintrag `prepend_locale: true` hinzufügen.
+Anschließend musst du über den Contao-Manager (»Systemwartung« > »Prod.-Cache erneuern«) oder über die Konsole 
+einmalig den Anwendungs-Cache leeren.
+{{% /expand %}}
+
+{{% expand "Meine Bilder werden im Frontend nicht angezeigt." %}}
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
{{% expand "Meine Bilder werden im Frontend nicht angezeigt, was kann ich machen?" %}}
```

File: docs/manual/faq/_index.de.md [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+im Hintergrund eine Prüfung durch. Sollte eine neue Version verfügbar sein, aktualisiert sich der Contao Manager selbst.
+{{% /expand %}}
+
+{{% expand "Was ist der Composer Resolver Cloud?" %}}
+Der [Composer Resolver Cloud](https://composer-resolver.cloud/) erlaubt die Installation von Composer-Abhängigkeiten 
+über den [Contao Manager](/de/installation/contao-manager/), selbst wenn dein Server nicht über genug Arbeitsspeicher verfügt.
+{{% /expand %}}
+
+{{% expand "Wie aktiviere ich den Contao Debug-Modus?" %}}
+Hier findest du alle Informationen zum Contao [Debug-Modus](/de/system/debug-modus/).
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
Du kannst den [Contao Debug-Modus](/de/system/debug-modus/) über das Backend aktivieren.
```

File: docs/manual/faq/_index.de.md [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+{{% /expand %}}
+
+{{% expand "Wo finde ich weitere Contao-Ressourcen?" %}}
+Hier erhälst du eine Übersicht zu weiteren [Contao-Ressourcen](https://contao.org/de/netzwerk.html).
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
Weiter Contao-Ressourcen findest du auf der [Projektwebseite](https://contao.org/de/netzwerk.html).
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]

**Comment by fkaminski** [comment]
@netzarbeiter  
Maybe we should better structure the faq with h3-Titles - for now.
However, there are many labeling options left

**Comment by netzarbeiter** [comment]
Three request are still open. If we go with the headline-tag, it should be h2 (There is only the maintitle with h1).

**Comment by fkaminski** [comment]
If merged - maybe we could then leave a hint/link in:
https://contao.org/de/faq.html

**Comment by netzarbeiter** [comment]
Thank you, @fkaminski

**Pull request merged**